### ISSUE TITLE
elements of the user themes are unskinnable/do not follow skin

### ISSUE THREAD

**Issue by ashydesu** [modernspring/retrospring#2]
this includes the purple circle behind the number of new notifications, 'success' and presumably other popups, the 'delete this question' confirmation screen and possibly others.

**Comment by SEAPUNK** [comment]
Yes, I know. I never bothered to completely finish the theming process, and I doubt I will for this software in particular.

**Comment by ashydesu** [comment]
so why not just pull working code from RS, for example?

**Comment by SEAPUNK** [comment]
All of the code that could be pulled from RS has already been pulled. Past that, it's a matter of making the [style override](https://github.com/modernspring/retrospring/blob/master/app/assets/stylesheets/ja-override.css.scss) play nice with the themes.

**Comment by ashydesu** [comment]
... or you could not use an override and fix the stylesheets to work with the colours you want instead

**Comment by SEAPUNK** [comment]
Coming back to my original comment:

>[...] and I doubt I will for this software in particular.
